packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -891,3 +891,49 @@ import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-i
   </DescriptionListGroup>
 </DescriptionList>;
 ```
+
+### Column fill
```

Comment:
@boaz0 can you move this example up so it comes after fluid horizontal? Currently it's under the auto-fit section and doesn't belong there.